pkg/deploy/api/v1/types.go

```diff
@@ -278,14 +278,23 @@ type DeploymentConfigSpec struct {
 
 // DeploymentConfigStatus represents the current deployment state.
 type DeploymentConfigStatus struct {
-	// LatestVersion is used to determine whether the current deployment associated with a DeploymentConfig
-	// is out of sync.
+	// LatestVersion is used to determine whether the current deployment associated with a deployment
+	// config is out of sync.
 	LatestVersion int `json:"latestVersion,omitempty"`
-	// Details are the reasons for the update to this deployment config.
-	// This could be based on a change made by the user or caused by an automatic trigger
+	// Details are the reasons for the update to this deployment config. This could be based on a
+	// change made by the user or caused by an automatic trigger.
 	Details *DeploymentDetails `json:"details,omitempty"`
-	// ObservedGeneration is the most recent generation observed by the controller.
+	// ObservedGeneration is the most recent generation observed by the deployment config controller.
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	// Replicas is the total number of pods targeted by this deployment config.
+	Replicas int32 `json:"replicas,omitempty"`
+	// UpdatedReplicas is the total number of non-terminated pods targeted by this deployment config
+	// that have the desired template spec.
+	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
```

Comment:
Every field is upstream already

On Thu, Jun 23, 2016 at 4:36 PM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/deploy/api/v1/types.go
> <https://github.com/openshift/origin/pull/6233#discussion_r68246188>:
>
> >  	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
> > +	// Replicas is the total number of pods targeted by this deployment config.
> > +	Replicas int32 `json:"replicas,omitempty"`
> > +	// UpdatedReplicas is the total number of non-terminated pods targeted by this deployment config
> > +	// that have the desired template spec.
> > +	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
>
> Which of these are not upstream?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/6233/files/23f2869e9501ebeb135cad392256f527d16b67a0#r68246188>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADuFf6JMAY9Jc3tBwTd-gsUEqEY4eKw5ks5qOpnogaJpZM4GxCXA>
> .
>